doc/man3/OSSL_PARAM.pod

```diff
@@ -40,11 +40,11 @@ suitable form for the internal structure of the object.
 
 =item * Request parameters of some object
 
-The caller (the I<requestor>) sets up the B<OSSL_PARAM> array and
+The caller (the I<requester>) sets up the B<OSSL_PARAM> array and
 calls some function (the I<responder>) that has intimate knowledge
 about the object, which can take the internal data of the object and
 copy (possibly convert) that to the memory prepared by the
-I<requestor> and pointed at with the B<OSSL_PARAM> I<data>.
```

Comment:
@levitte The _requester_ vs. _requestor_ issue has been moved from #20910 to this #20924 PR. Based on https://github.com/openssl/openssl/pull/20910#discussion_r1188772911 and https://github.com/codespell-project/codespell/pull/2845#issuecomment-1541905325, would you agree with this change?